### PR TITLE
NewRelic deployment hook

### PR DESCRIPTION
**Pull request by joaodaher**
Add NewRelic deployment hook using [its REST API](https://rpm.newrelic.com/explore/application_deployments/create).

### CODE REVIEW

File: hook/newrelic.go [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+	}
+	if err := s.createDeploy(httpClient, deploys[0]); err != nil {
+		return err
+	}
+	return nil
+}
+
+func (s *NewRelic) createDeploy(httpClient *http.Client, deploy types.Deploy) error {
+	data := []byte(`
+	{ "deployment": {
```
Comment (lucasgomide) [inline hook/newrelic.go]: Can you improve data's formatting, for better legibility ?

**Review by lucasgomide — COMMENTED** [review]
Tks for your contribution (:

File: hook/newrelic.go (hunk 1 above)
Comment (joaodaher) [reply hook/newrelic.go]: Sure, fixed!

**Review by lucasgomide — APPROVED** [review]
tks for you contribution.

**Pull request merged**